packages/react-reconciler/src/__tests__/ReactSuspensePlaceholder-test.internal.js

```diff
@@ -403,10 +403,21 @@ describe('ReactSuspensePlaceholder', () => {
         expect(onRender).toHaveBeenCalledTimes(2);
 
         // The suspense update should only show the "Loading..." Fallback.
-        // Both durations should include 10ms spent rendering Fallback
-        // plus the 8ms rendering the (hidden) components.
-        expect(onRender.mock.calls[1][2]).toBe(18);
-        expect(onRender.mock.calls[1][3]).toBe(18);
+        // The actual duration should include 10ms spent rendering Fallback,
+        // plus the 8ms render all of the hidden, suspended subtree.
+        // Note from Andrew to Brian: I don't fully understand why this one
+        // diverges, but I checked and it matches the times we get when
+        // we run this same test in Concurrent Mode.
```

Comment:
cc: @bvaughn Does this look ok? I don't remember if the legacy behavior diverged intentionally or if it was an implementation compromise. Naively, it seems fine for it to match the numbers in Concurrent Mode but I trust your judgment more than mine.

Reply:
Oh man. This reaches back a while...

I *think* the 18/15 numbers are more accurate, so if the *new* behavior also produces those for sync roots- that seems nice.